channels/ru.m3u

```diff
@@ -761,8 +715,6 @@ http://ott-cdn.ucom.am/s52/04.m3u8
 http://spacetv.in/stream/DVB2D6AE8CU/138.m3u8
 #EXTINF:-1 tvg-id="STS.ru" tvg-name="СТС" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/pJNWVAz.png" group-title="",СТС
 https://sc.id-tv.kz/STS.m3u8
-#EXTINF:-1 tvg-id="STSPlus4.ru" tvg-name="СТС +4" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",СТС +4 (576p)
-http://bar-timeshift-inet.ll-bar.zsttk.ru:8080/sts/playlist.m3u8
```

Comment:
Do we care about regional variants with time shifts? If so, I can make this playlist to consist of 1000+ channels

Reply:
Well, not quite sure, unless that's official channels being intentionally streamed like it.

Reply:
I don't see anything wrong with that, especially if the channel has its own program like this one: http://www.vsetv.com/schedule_channel_539_week.html